test/unit/pdf_find_controller_spec.js

```diff
@@ -780,4 +780,38 @@ describe("pdf_find_controller", function () {
       },
     });
   });
+
+  it("performs a search in a text containing combining diacritics", async function () {
```

Comment:
Given that this is a linked test-case, you need the following here as well: https://github.com/mozilla/pdf.js/blob/d11cae6f2768bea382dc21873ef4c608bc036e53/test/unit/pdf_find_controller_spec.js#L645-L647